packages/demos/package.json

```diff
@@ -8,11 +8,11 @@
   },
   "scripts": {
     "build": "build-storybook -c .storybook -o dist",
+    "chromatic": "chromatic --project-token=$CHROMATIC_TOKEN",
     "clean": "rimraf dist",
     "demo": "yarn storybook",
     "prebuild": "yarn run clean",
     "start": "yarn storybook",
-    "storybook": "start-storybook -p 9999",
-    "chromatic": "chromatic --project-token=$CHROMATIC_TOKEN"
+    "storybook": "start-storybook -p 9999"
```

Comment:
do you mind using ports that are one next to each other instead?

Reply:
like 1001, 1002, 1003?

I guess it will work too.

Could we somehow parametrize this though, so we could use it on the storybook config?

Reply:
OK, will do!